sdk/core/azure-core/src/main/java/com/azure/android/core/http/interceptor/LogOptions.java

```diff
@@ -146,17 +159,26 @@ public String getApplicationId() {
      */
     public LogOptions setApplicationId(final String applicationId) {
```

Comment:
Actually why is this in LogOptions?  AFAIK the application ID is used only for the user agent interceptor.

Reply:
@bsiegel This was originally taken from the Java SDK. Could it be that they wanted to include the Application ID in the logs?

Reply:
Anyways, I will remove this from here for now and ask the Java team the reasoning behind this. If we see value in having it we can add it later.

Reply:
Sounds good, as far as I know it's only used in that one place. If we find a use for it in logging we can always add it back.